### ISSUE TITLE
Cupertino Picker error

### ISSUE THREAD

**Issue by NTMS2017** [flutter/flutter#37166]
Cupertino Picker error, any idea?

 ```
(28557): A SemanticsNode with action "increase" needs to be annotated with either both "value" and
I/flutter (28557): "increasedValue" or neither
I/flutter (28557): 'package:flutter/src/semantics/semantics.dart':
I/flutter (28557): Failed assertion: line 1770 pos 7: '!_canPerformAction(SemanticsAction.increase) || (_value == '')
I/flutter (28557): == (_increasedValue == '')'
I/flutter (28557): Either the assertion indicates an error in the framework itself, or we should provide substantially
I/flutter (28557): more information in this error message to help you determine and fix the underlying cause.
I/flutter (28557): In either case, please report this assertion by filing a bug on GitHub:
I/flutter (28557):   https://github.com/flutter/flutter/issues/new?template=BUG.md
I/flutter (28557): When the exception was thrown, this was the stack:
I/flutter (28557): #2      SemanticsNode.updateWith (package:flutter/src/semantics/semantics.dart:1770:7)
I/flutter (28557): #3      _RenderCupertinoPickerSemantics.assembleSemanticsNode (package:flutter/src/cupertino/picker.dart:481:10)
I/flutter (28557): #4      _SwitchableSemanticsFragment.compileChildren (package:flutter/src/rendering/object.dart:3460:13)
```

**Comment by HansMuller** [comment]
It would help if you could provide the output of `flutter doctor` and a small self-contained app that demonstrates the problem. Note also: https://gist.github.com/ is a useful way to share the code.

**Comment by NTMS2017** [comment]
I will used Material, and I will test it again using Cupertino and open new issue if problem still exist. Thanks

**Comment by DrFrancky** [comment]
This is still an issue. 
If one of the values in the List is '' then this error will pop up.

**Comment by dvdmmc** [comment]
I got this error when I tried to mix Text() and Icon() children in the same CupertinoPicker.  Adding a semanticLabel to the Icons (good practice regardless) stopped the assertion from failing
